From 934cff22fcf28da1e5d5a0ea4d9298d8c6c4b310 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:48:36 -0400
Subject: [misc] mei: Add include of <linux/irqreturn.h>

Message-id: <20140806034836.1438.23505.stgit@gir.stowe>
Patchwork-id: 87451
O-Subject: [RHEL7.1 PATCH 42/69] mei: Add include of <linux/irqreturn.h>
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: 81ec5502a0c87eaae424a6c733a9da2bd0b4d866

commit 81ec5502a0c87eaae424a6c733a9da2bd0b4d866
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Feb 14 14:06:14 2014 -0700

    mei: Add include of <linux/irqreturn.h>

    We currently include <linux/irqreturn.h> in <linux/pci.h>, but I'm about to
    remove that from linux/pci.h, so add explicit includes where needed.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Acked-by: Tomas Winkler <tomas.winkler@intel.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/hw-me.h b/drivers/misc/mei/hw-me.h
index 80bd829..893d511 100644
--- a/drivers/misc/mei/hw-me.h
+++ b/drivers/misc/mei/hw-me.h
@@ -20,6 +20,7 @@
 #define _MEI_INTERFACE_H_
 
 #include <linux/mei.h>
+#include <linux/irqreturn.h>
 #include "mei_dev.h"
 #include "client.h"
 
-- 
1.7.1